docs/core/docker/build-container.md

```diff
@@ -22,30 +22,19 @@ You'll learn to:
 
 You'll understand the Docker container build and deploy tasks for a .NET Core application. The *Docker platform* uses the *Docker engine* to quickly build and package apps as *Docker images*. These images are written in the *Dockerfile* format to be deployed and run in a layered container.
 
+> [!TIP]
```

Comment:
I know. It's confusing for the simple fact people do land here and they do read through it even though they have an ASP.NET Core app. So the thought on this is to

1. Point them over the ASP.NET tutorial; it will help them consider ASP.NET specific stuff.
2. If they keep using this one, by specifying the ASP.NET Core runtime, they may succeed in containerizing their app. (reducing problems)
3. Keep them in the context and thought of containerizing a console app (even though the other runtime is selected).